src/Microsoft.DotNet.Build.Tasks/PackageFiles/launchSettings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "profiles": {
+        ".NET Core xUnit Console": {
+            "commandName": "Executable",
+            "executablePath": "$(BinDir)testhost/netcoreapp-$(_bc_OSGroup)-$(ConfigurationGroup)-$(ArchGroup)/dotnet.exe",
+            "commandLineArgs": "xunit.console.dll $(XunitArguments) -wait -parallel none",
+            "workingDirectory": "$(TestWorkingDir)$(MSBuildProjectName)/netcoreapp-$(_bc_OSGroup)-$(ConfigurationGroup)-$(ArchGroup)/"
+        },
+        ".NET Framework xUnit Console": {
+            "commandName": "Executable",
+            "executablePath": "$(TestWorkingDir)$(MSBuildProjectName)/netfx-Windows_NT-$(ConfigurationGroup)-$(ArchGroup)/xunit.console.exe",
+            "commandLineArgs": "$(XunitArguments) -wait -parallel none",
+            "workingDirectory": "$(TestWorkingDir)$(MSBuildProjectName)/netfx-Windows_NT-$(ConfigurationGroup)-$(ArchGroup)/",
+            "environmentVariables": {
+                "DEVPATH": "$(BinDir)runtime/netfx-Windows_NT-$(ConfigurationGroup)-$(ArchGroup)/"
```

Comment:
Doesn't matter much since in netfx we binplace the same binaries in bin/runtime and bin/testhost, but to be consistent should we use testhost instead? That is what is used by Helix and RunTests.cmd since %RUNTIME_PATH% is set to that as that is the first argument passed to the script.